chainer/training/extensions/parameter_statistics.py

```diff
@@ -42,19 +43,21 @@ class ParameterStatistics(extension.Extension):
     default_name = 'parameter_statistics'
     priority = extension.PRIORITY_WRITER
 
+    # avoid computations involving NaNs, e.g. when initial gradients are NaNs
+    check_nan_params = False
```

Comment:
How about making it `__init__` argument?
Also write documentation instead of comment.

Reply:
Okay, fixed!